.github/workflows/ci.yaml

```diff
@@ -111,17 +110,6 @@ jobs:
             ghcr.io/elifesciences/enhanced-preprints-client:latest
             ghcr.io/elifesciences/enhanced-preprints-client:${{ github.sha }}
             ghcr.io/elifesciences/enhanced-preprints-client:master-${{ steps.commit_sha.outputs.commit_sha }}-${{ steps.date.outputs.date }}
-      - name: Build and push storybook image
```

Comment:
This just removes storybook build (which isn't deployed anywhere anymore) and combined with the changes to the Dockerfile should speed up build times a lot